cmd/brew-services.rb

```diff
@@ -155,9 +160,18 @@ def list
       puts format("#{Tty.white}%-#{longest_name}.#{longest_name}s %-7.7s %-#{longest_user}.#{longest_user}s %s#{Tty.reset}",
                   "Name", "Status", "User", "Plist")
       formulae.each do |formula|
+        status = case formula[:status]
+                 when :started
```

Comment:
Happy to do that if that's what you prefer, I'm just following the community style guide here (https://github.com/bbatsov/ruby-style-guide#indent-conditional-assignment)